addons/mail/models/mail_mail.py

```diff
@@ -171,7 +205,7 @@ def _send_prepare_values(self, partner=None):
         body = self._send_prepare_body()
         body_alternative = tools.html2plaintext(body)
         if partner:
-            email_to = [formataddr((partner.name or 'False', partner.email or 'False'))]
+            email_to = partner._send_prepare_partner_adress()
```

Comment:
it is mainly because I'm using the same formatting to identify failed email in postprocess, if we decide to change slightly this formating and forget to change it everywhere it could create issue. Don't you think it's cleaner this way?